client/src/player.rs

```diff
@@ -64,16 +64,18 @@ pub struct PlayerController {
     scroll: f32,
     x_sensitivity: f32,
     y_sensitivity: f32,
+    scroll_sensitivity: f32,
 }
 
 impl PlayerController {
-    pub fn new(x_sensitivity: f32, y_sensitivity: f32) -> Self {
+    pub fn new(x_sensitivity: f32, y_sensitivity: f32, scroll_sensitivity:f32) -> Self {
```

Comment:
> I think the PlayerController on main only has 2 arguments in the constructor.

I added a new argument for scroll sensitivity to the constructor

Reply:
Ah, I see it now. Sorry about that.